tools/test-js-with-node

```diff
@@ -160,7 +160,7 @@ EXEMPT_FILES = make_set(
         "web/src/navbar_alerts.ts",
         "web/src/navbar_help_menu.ts",
         "web/src/navbar_menus.js",
-        "web/src/navigate.ts",
+        "web/src/navigate.js",
```

Comment:
This may be avoidable with an eslint check but it's such a trivial file that seems OK.